test/nodeMock/app.js

```diff
@@ -117,26 +117,67 @@ if (args.standalone){
 
 function setUpcaseConfig() {
     const { getTestJurisdiction }  = require('../ngIntegration/mockData/ccdCaseMock');
-    mockInstance.onGet('/data/internal/case-types/:jurisdiction/event-triggers/:caseType', (req, res) => {
-        const caseEventConfig = getTestJurisdiction();
+    mockInstance.onGet('/data/internal/cases/:caseid', (req, res) => {
+        
+        res.send(caseDetailsLabelShowCondition().getCase());
+    });
 
-        let multiSelectListField = caseEventConfig.getCaseFieldConfig("MultiSelectListField");
-        // const page1 = caseEventConfig.getWizardPageConfig("testPage1");
-        // page1.label = "For demo 123456"
+}
 
-        const textField0 = caseEventConfig.getCaseFieldConfig("TextField0");
-        textField0.display_context = "MANDATORY";
+function caseDetailsLabelShowCondition(){
+    const caseDetail = new CCDCaseDetails("Mock Label show cosndition case type"); 
```

Comment:
Typo: `cosndition` - suspect this should be `condition`.

Reply:
Another one for @sreekanthpuligadda.

Reply:
addressed.